replayer/build.gradle

```diff
@@ -0,0 +1,51 @@
+apply plugin: 'com.android.application'
+apply plugin: 'io.fabric'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+apply from: '../gradle/generate-token.gradle'
+apply from: "../gradle/utils.gradle"
+apply from: "../gradle/ktlint.gradle"
+
+android {
+    buildToolsVersion build_tools_version
+    compileSdkVersion compile_sdk_version
+
+    defaultConfig {
+        applicationId "com.mapbox.vision.replayer"
+        minSdkVersion min_sdk_version
+        targetSdkVersion target_sdk_version
+        versionCode 1
+        versionName "1.0"
```

Comment:
should we use same version as for teaser?

Reply:
I think no, it's a separate apps, and we don't release Replayer